about.html

```diff
@@ -0,0 +1,169 @@
+      <li class="top-nav">
+        <a class="nav-link" href="#">Join</a>
+      </li>
+      <li class="top-nav">
+        <a class="nav-link" href="#">Sponsor</a>
+      </li>
+      <li class="top-nav">
+        <a class="nav-link" href="#">News</a>
+      </li>
+      <li class="top-nav">
+        <a class="nav-link campain" href="#">CC Campaign</a>
+      </li>
+    </ul>
+  </div>
+</nav>
+
+    <nav class="desktop-nav">
+      <a href="index.html"> <img src="images/Zambia_BBall-Association.jpg" alt="logo zambia basketball" class="logo-image3"> </a>
+      <ul class="desk-menu">
+            <li> 
+              <a class=" nav-about active" href="index.html">Home</a>
+            </li>
+            <li>
+              <a  href="#">Program</a>
+            </li>
+            <li> 
+              <a  href="#">Join</a>
+            </li>
+            <li>
+              <a  href="#">Sponsor</a>
+            </li>
+            <li>
+              <a  href="#">News</a>
+            </li>
+            <li>
+              <a class="campain" href="#"> CC Campaign</a>
+            </li>
+       
+          </ul>
+      
+        </nav>
+      </div>
+  </header>
+
+  <section class="about-container">
+    <h1 class="about-header">Zambia basketball all star weekend 2022</h1>
+    <p class="more-details-about">
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Accusantium expedita 
+      dolore pariatur vel, architecto ipsam ut reprehenderit adipisci atque molestiae reiciendis quas, 
+      culpa beatae porro ratione laudantium magnam! Eum animi quos nisi, aperiam cupiditate dignissimos ducimus 
+      architecto illum assumenda ab nemo? Necessitatibus fuga labore doloribus repellendus molestiae, minima vitae atque.
+    </p>
+    <p class="contact">
+      Please contact us per Email for any further questions <br>about CC Global Summit 2021! <br>
+  </p>
+  <p class="email-link">zambiabasketall@zba.co.za</p>
+  
+  </section>
+  <section class="about-logo-area">
+    <h2 class="all-start-text">Zambia Basketall All Star  Weekend Logo</h2>
+    <hr>
+    <p class="details-about-logo">Lorem ipsum dolor sit amet consectetur adipisicing elit. 
+      Illo eum dignissimos quasi <br> necessitatibus suscipit sit laborum, 
+      possimus provident fugit modi.
+    </p>
+    <img src="images/Zambia_BBall-Association.jpg" alt="zba logo" class="logo-zba">
+  </section>
+  <hr class="all-start-line">
+  <section class="paster-all-start">
+    <h3 class="all-star_past-event"> See the past ZBA Weekend</h3>
+    <hr>
+    <p class="past-event">
+      Take a look at the last two ZBA all weekend wich took place in Livingstone and Ndola city.
+    </p>
+    <div class="event-wrapper">
+      <article class="event1">
+          <h2>2020</h2>
+          <p>All start weekend Livingstone, Zambia</p>
+      </article>
+      <article class="events2">
+          <h2>2021</h2>
+          <p>All star weekend ndola city,Zambia</p>
+      </article>
+  </div>
```

Comment:
- [ ] [OPTIONAL] Kindly ensure that your design looks more like the original design provided. Please ensure you add images to the two cards and an overlay.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![Screenshot from 2022-04-14 22-15-28](https://user-images.githubusercontent.com/58692590/163478699-45e31620-9b58-48f4-b95a-4ca480db1042.png) | ![Screenshot from 2022-04-14 22-15-43](https://user-images.githubusercontent.com/58692590/163478767-1d93d1a1-5478-47c4-bb84-70fbcde2913e.png)